[BUG] openjdk:21-jdk-slim 이미지 사용 불가

## Description
Dockerfile에서 사용 중인 `openjdk:21-jdk-slim` 이미지가 Docker Hub에서 pull되지 않아 CD 과정에서 빌드 및 배포가 실패했습니다. 

알아본 결과, 최근 Docker Hub - OpenJDK 깃허브 레포지토리에 특정 PR이 반영되며 몇몇 이미지가 더 이상 제공되지 않는 듯합니다.
- [PR](https://github.com/docker-library/openjdk/pull/550)
- [ISSUE](https://github.com/docker-library/openjdk/issues/551)

## Logs / Error
```
Dockerfile:1
--------------------
   1 | >>> FROM openjdk:21-jdk-slim
   2 |     
   3 |     RUN addgroup --system app && adduser --system --ingroup app app
--------------------
ERROR: failed to build: failed to solve: openjdk:21-jdk-slim: failed to resolve source metadata for docker.io/library/openjdk:21-jdk-slim: docker.io/library/openjdk:21-jdk-slim: not found
Error: Process completed with exit code 1.
```

## To-do
- [ ] Dockerfile의 JDK 이미지 수정

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM openjdk:21-jdk-slim
+FROM eclipse-temurin:21-jdk-jammy
 
 RUN addgroup --system app && adduser --system --ingroup app app
 

